.github/workflows/ci.yml

```diff
@@ -126,7 +126,9 @@ jobs:
             MODULES_MAVEN_PARAM="-pl ${MODULES_ARG}"
           fi
 
-          mvn -fae -V -B -s .github/mvn-settings.xml clean verify -Dall-modules -Dinclude.quarkus-cli-tests -Dts.quarkus.cli.cmd="${PWD}/quarkus-dev-cli" $MODULES_MAVEN_PARAM -am
+          mvn -fae -V -B -s .github/mvn-settings.xml clean verify -Dquarkus-platform-version-2.8.3-or-higher="true" \
```

Comment:
I've been thinking about this and it seems to me that it might be easier to invert the condition: enable the bean and related tests unless Quarkus version is <2.8.3.

It would require changes in `NinetyNineBottlesOfBeerResource`:
```java
@RegisterForReflection
@UnlessBuildProperty(name = QUARKUS_PLATFORM_VERSION_LESS_THAN_2_8_3, stringValue = QUARKUS_PLATFORM_VERSION_LESS_THAN_2_8_3_VAL, enableIfMissing = true)
@Path(NinetyNineBottlesOfBeerResource.PATH)
public class NinetyNineBottlesOfBeerResource {

    public static final String QUARKUS_PLATFORM_VERSION_LESS_THAN_2_8_3 = "quarkus-platform-version-less-than-2.8.3";
    public static final String QUARKUS_PLATFORM_VERSION_LESS_THAN_2_8_3_VAL = "true";
    ...
}
```
and at `HttpAdvancedReactiveIT#uriPathTemplate`:
```java
    @DisabledIfSystemProperty(matches = QUARKUS_PLATFORM_VERSION_LESS_THAN_2_8_3_VAL, named = QUARKUS_PLATFORM_VERSION_LESS_THAN_2_8_3, disabledReason = TEST_REQUIRES_AT_LEAST_2_8_3)
    @DisplayName("JAX-RS URI path template test")
    @Test
    public void uriPathTemplate() {
        ...
    }
```

That way, we don't need to add any properties anywhere:
- `2.7` branch (and anything older) does not containt the bean/tests.
- `main` (and any future branches) will have the bean/tests enabled by default.

And in case we forget to add the property somewhere (if we backport the tests, or e.g. when running the TS with `-Dquarkus.platform.version=2.8.1.Final`), we will at least get a visible buildtime error.

WDYT?

Reply:
Absolutely, that's much better. Done, thanks for the suggestion.